generators/app/templates/src/FOOBAR/package.json

```diff
@@ -6,63 +6,66 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --prod && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
-    "build-dev": "ng build --source-map && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
-    "swagger-generate": "del-cli ./ClientApp/app/shared/models/api && mkdirp ./ClientApp/app/shared/models/api && swagger-angular-generator --no-store -s ../swagger/spec.json -d ./ClientApp/app/shared/models/api && for directory in common controllers store; do del-cli ./ClientApp/app/shared/models/api/${directory}; done",
-    "swagger-compare": "swagger-diff ../swagger/oldSpec.json ../swagger/newSpec.json",
+    "build:dev": "ng build --source-map && del-cli ./wwwroot/index.html && move-cli ./wwwroot/manifest.webmanifest ./wwwroot/manifest.json",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "compodoc": "./node_modules/.bin/compodoc -p ClientApp/tsconfig.app.json -n \"TITLEFOO documentation\""
+    "generate": "swagger-angular-generator --no-store -s ../swagger/spec.json -d ClientApp/app/models/api",
```

Comment:
Wat is het verschil tussen `generate` en `swagger:generate`? Is `generate` nodig?

Reply:
generate is idd niet meer nodig, is weg nu.